### PR TITLE
feat(tickets): add AI Training product

### PR DESCRIPTION
**Pull request by varun257**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-7029
| License          | BSD 3-Clause

## Description

Add ai training translation in ticket

### CODE REVIEW

**Review by Heiiji — DISMISSED** [review]

File: packages/manager/modules/support/src/tickets/new-ticket/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -98,6 +98,7 @@
   "ovhManagerSupport_new_serviceType_saas_csp_2": "Microsoft 365 et Office 365 Reseller",
   "ovhManagerSupport_new_serviceType_horizon_view": "Horizon View",
   "ovhManagerSupport_new_serviceType_other": "Autre demande",
+  "ovhManagerSupport_new_serviceType_ai_training_job": "Ai Training",
```
Comment (antleblanc) [inline packages/manager/modules/support/src/tickets/new-ticket/translations/Messages_fr_FR.json:101]: Shouldn't be:

```suggestion
  "ovhManagerSupport_new_serviceType_ai_training_job": "AI Training",
```

see: https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/sidebar/sidebar.constant.js#L134

**Review by antleblanc — CHANGES_REQUESTED** [review]

**Comment by antleblanc** [comment]
Rebasing on `master` branch and fixing both `fr_FR` and `fr_CA` translation files (`s/Ai/AI/g`).

**Comment by antleblanc** [comment]
Rebased successfully on `master` branch.

File: packages/manager/modules/support/src/tickets/new-ticket/translations/Messages_fr_FR.json (hunk 1 above)
Comment (antleblanc) [reply packages/manager/modules/support/src/tickets/new-ticket/translations/Messages_fr_FR.json]: Fixed via the rebase on `master` branch.

**Review by antleblanc — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**